docs/item.rst

```diff
@@ -0,0 +1,200 @@
+    implicit filters of the selected main action.
+
+``-a, --include-artifacts``
+    Include artifacts in the item list. Regardless of this setting, artifacts
+    are never dumped or melted.
+
+``-i, --inside <burrow>``
+    Only include items inside the given burrow.
+
+``-o, --outside <burrow>``
+    Only include items outside the given burrow.
+
+``-r, --reachable``
+    Only include items reachable by one of your citizens.
+
+``-u, --unreachable``
+    Only include items not reachable by any of your citizens.
+
+``-t, --type <string>``
+    Filter by item type (e.g., BOULDER, CORPSE, ...). Also accepts lower case
+    spelling (e.g. "corpse"). Use ``:lua @df.item_type`` to get the list of all
+    item types.
+
+``-m, --material <string>``
+    Filter by material the item is made out of (e.g., "iron").
+
+``-c, --mat-category <string>``
+    Filter by material category of the material item is made out of (e.g.,
+    "metal"). Use ``:lua df.dfhack_material_category`` to get a list of all
+    material categories.
+
+``-d, --description <pattern>``
+    Filter by item description (singular form without stack sizes). The
+    ``pattern`` is a Lua pattern
+    (cf. https://www.lua.org/manual/5.4/manual.html#6.4.1), so "cave spider
+    silk" will match both "cave spider silk web" as well as "cave spider silk
+    cloth". Use ``^pattern$`` to match the entire description.
+
+``-w, --min-wear <integer>``
+    Only include items whose wear/damage level is at least ``integer``. Useful
+    values are 0 (pristine) to 3 (XX).
+
+``-W, --max-wear <integer>``
+    Only include items whose wear/damage level is at most ``integer``. Useful
+    values are 0 (pristine) to 3 (XX).
+
+``-q, --min-quality <integer>``
+    Only include items whose quality level is at least ``integer``. Useful
+    values are 0 (ordinary) to 5 (masterwork).
+
+``-Q, --max-quality <integer>``
+    Only include items whose quality level is at most ``integer``. Useful
+    values are 0 (ordinary) to 5 (masterwork).
+
+``--stockpiled``
+    Only include items that are in stockpiles. Does not include empty bins,
+    barrels, and wheelbarrows assigned as storage and transport for stockpiles.
+
+``--scattered``
+    Opposite of ``--stockpiled``
+
+``--forbidden``
+    Only include forbidden items.
+
+``--melting``
+    Only include items designated for melting.
+
+``--dumping``
+    Only include items designated for dumping.
+
+``--visible``
+    Only include visible items (i.e., ignore hidden items).
+
+``--hidden``
+    Only include hidden items.
+
+
+API
+---
+
+The item script can be called programmatically by other scripts, either via the
+commandline interface with ``dfhack.run_script()`` or via the API functions
+defined in :source-scripts:`item.lua`, available from the return value of
+``reqscript('item')``:
+
+* ``item.execute(action, conditions, options)``
+* ``item.executeWithPrinting(action, conditions, options)``
```

Comment:
```suggestion
``reqscript('item')``::

    local itemtools = reqscript('item')
    itemtools.execute(action, conditions, options)
    itemtools.executeWithPrinting(action, conditions, options)
```

Reply:
I'm reluctant to call the variable that holds the `reqscript` return value "item" in the example since that is used in so many scripts as a local reference to an actual item.